features/order/managing_orders/placing_an_order_with_shipment.feature

```diff
@@ -18,4 +18,4 @@ Feature: Shipments are in the state "ready" after checkout
     @ui
     Scenario: Checking shipment state of a placed order
         When I view the summary of the order "#00000666"
-        Then it should have shipment in state "Ready"
+        Then it should have shipment state ready
```

Comment:
Why changing this? The above step checks where there is a shipment in state "Ready", I think we don't need to hide `"Ready"`, unless we would change the step to `it should have shipment ready to be processed` or something similar. But that's a separate topic for RFC.